scripts/drone/events/release.star

```diff
@@ -131,146 +89,10 @@ def release_npm_packages_step():
         "commands": ["./bin/build artifacts npm release --tag ${DRONE_TAG}"],
     }
 
-def oss_pipelines(ver_mode = ver_mode, trigger = release_trigger):
```

Comment:
this wasn't actually being used ;)

Reply:
Probably some left over from the enterprise move, it was hard to tell what's being used so some is left over.